app/Policies/PostPolicy.php

```diff
@@ -28,7 +29,7 @@ public function viewAll($user, Post $post)
      * @param  Rogue\Models\Post $post
      * @return bool
      */
-    public function show($user, Post $post)
+    public function show($user = null, Post $post)
```

Comment:
@placeholder... fill in details regarding this bug fix.

Reply:
I think you can typehint this to `Illuminate\Contracts\Auth\Authenticatable` (as `?Authenticatable`) which both our local users (by [extending](https://github.com/DoSomething/rogue/blob/8bcfe954aa20233afe0377cedd45d5a03bb218de/app/Models/User.php#L7) [this class](https://github.com/laravel/framework/blob/78eb4dabcc03e189620c16f436358d41d31ae11f/src/Illuminate/Foundation/Auth/User.php#L5)) and remote users ([here](https://github.com/DoSomething/gateway/blob/0ae0f829e96e0e674c4906cc3444ec57eacf5b11/src/Server/RemoteUser.php#L6)) share.

Reply:
Ah true! Ok, will give this a shot!